Src/Generic/Extensions/UIControl+DelayedTouchCancel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UIControl+DelayedTouchCancel.swift
+//  Ugglan
+//
+//  Created by Gustaf Gunér on 2019-05-29.
+//
+
+import Flow
+import Foundation
+import UIKit
+
+extension UIControl {
+    func delayedTouchCancel(delay: Double = 0.2) -> Signal<Bool> {
```

Comment:
Usually time is presented with the TimeInterval type, which is a typealias for a Double, but still 😄

Reply:
The more descriptive the better 😄